<launch>
	<node name="cv_camera0" pkg="cv_camera" type="cv_camera_node" output="screen" >
		<param name="device_id" value="0" />
		<param name="frame_id" value="cv_camera_0" />
		<param name="image_width" value="640" />
		<param name="image_height" value="480" />
	</node>
	<node name="cv_camera1" pkg="cv_camera" type="cv_camera_node" output="screen" >
		<param name="device_id" value="1" />
		<param name="frame_id" value="cv_camera_1" />
		<param name="image_width" value="640" />
		<param name="image_height" value="480" />
	</node>

	<node name="image_converter0" pkg="cvTest" type="image_converter" >
		<remap from="/cv_camera/image_raw" to="cv_camera0/image_raw" />
	</node>
	<node name="image_converter1" pkg="cvTest" type="image_converter" >
		<remap from="/cv_camera/image_raw" to="cv_camera1/image_raw" />
	</node>
</launch>
